lib/generator.js

```diff
@@ -1281,8 +1281,10 @@
                                     (targetWidth ? (targetWidth / paddedInputWidth) : targetScaleY)),
 
             // Effects are not scaled when the transformation is non-uniform
-            effectsScaled       =
-                (paddedOutputWidthFloat / paddedInputWidth) === (paddedOutputHeightFloat / paddedInputHeight),
+            paddedFloatRatioDiff = Math.abs(paddedOutputWidthFloat / paddedInputWidth -
+                                            paddedOutputHeightFloat / paddedInputHeight),
+            effectsScaled       = (targetWidth || targetHeight) ? paddedFloatRatioDiff < Number.EPSILON :
```

Comment:
Err, did I perhaps not get the memo that we upgraded to node v0.12? Looks like Number.EPSILON is ES6, and that's on in v0.12? Or, did we do something else to enable ES6?

Seems like checking if everything works in v0.12 would be a non-trivial QE effort. is that happening?